website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -115,6 +121,8 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 - **New:** Enabled `where` filters  on dimensions (included in saved queries) to use the cache during query time. This means you can now dynamically filter your dashboards without losing the performance benefits of caching. Refer to [caching](/docs/use-dbt-semantic-layer/sl-cache#result-caching) for more information.
 - **Enhancement:** In [Google Sheets](/docs/cloud-integrations/semantic-layer/gsheets), we added information icons and descriptions to metrics and dimensions options in the Query Builder menu. Click on the **Info** icon button to view a description of the metric or dimension. Available in the following Query Builder menu sections: metric, group by, where, saved selections, and saved queries.
 - **Enhancement:** In [Google Sheets](/docs/cloud-integrations/semantic-layer/gsheets), you can now apply granularity to all time dimensions, not just metric time. This update uses our [APIs](/docs/dbt-cloud-apis/sl-api-overview) to support granularity selection on any chosen time dimension.
+- **Enhancement**: MetricFlow times spine warnings now prompt users to configure missing or small-grain-time spines. For multiple time spines per granularity, an error message will be displayed
```

Comment:
```suggestion
- **Enhancement**: MetricFlow time spine warnings now prompt users to configure missing or small-grain-time spines. For multiple time spines per granularity, an error message will be displayed.
```

Reply:
Hiya Mirna

I removed the plural s from time as it read a bit odd in my head and out loud and added a full stop at the end of the sentence.

Kind Regards
Natalie